core/core.go

```diff
@@ -242,14 +243,21 @@ func authok(user *auth.User, nodeuser *nodestore.User, node *nodestore.Node) boo
 }
 
 // GetFile gets the file from a node. Returns NoBlobError and UnauthorizedError.
```

Comment:
Might want to update that it also reports IllegalInputErrors

Reply:
Good eye, fixed